frontend/src/metabase-lib/queries/NativeQuery.ts

```diff
@@ -154,15 +154,15 @@ export default class NativeQuery extends AtomicQuery {
   /**
    * Returns true if the database metadata (or lack thererof indicates the user can modify and run this query
    */
-  readOnly() {
-    const database = this.database();
-    return !database || database.native_permissions !== "write";
+  readOnly(): boolean {
```

Comment:
I am guessing the comment should be updated too?

Reply:
Fixed

Reply:
fwiw, i don't think we really need that comment anymore, it's clear it's the opposite of `isEditable` from the method itself